src/objects/Attribute__c.object

```diff
@@ -114,6 +149,31 @@
                     <default>false</default>
                     <label>Other</label>
                 </value>
+                <value>
+                    <fullName>Parent in Military </fullName>
```

Comment:
There's a space at the end of this, as well as farther down below. I don't know if that will affect anything, but I think we should get rid of it just in case it would?

Reply:
The re-pull fixed this for me.